packages/flutter/lib/src/services/platform_messages.dart

```diff
@@ -6,95 +6,50 @@ import 'dart:async';
 import 'dart:typed_data';
 import 'dart:ui' as ui;
 
-import 'package:flutter/foundation.dart';
-
+import 'binary_messenger.dart';
 import 'binding.dart' show ServicesBinding;
 import 'platform_channel.dart';
 
-typedef _MessageHandler = Future<ByteData> Function(ByteData message);
-
 /// Sends binary messages to and receives binary messages from platform plugins.
 ///
```

Comment:
probably worth adding something here about how one should use `defaultBinaryMessenger` now instead.

Reply:
Done.